bench-exchange/README.md

```diff
@@ -59,76 +51,97 @@ matching trade orders.  All the transactions can execute concurrently.
-    swap requirements.  A trade swap may not wholly satisfy one or both of the
-    trade orders in which case the trade orders are adjusted appropriately.  As
-    long as the swap requirements are met there will be an exchange of tokens
-    between accounts.  Any price spread is deposited into the Swapper's profit
-    account.  All trade swaps are recorded in a new account for posterity.
+  - The difference between the two matching orders. The spread is the
+    profit of the matcher initiating the swap request.
+- Execution Conditions
+  - Conditions which when met, result in a successful trade being carried out.
+    - Orders in question must complement each other in offer/accept assets
+    - Orders must be valid and active (not expired/cancelled
+
+- Match request
+  - A request submitted to the exchange to carry out the matching process
+  - Requires a set of valid, active orders that meet execution conditions
+- Match
+  - Initiated via the sending of a match request to the exchange, a basic match
+    requires two valid, complementary orders and if successful, results in a Trade being produced.
+  - Requested by a 3rd party matcher under normal circumstances, but may be carried out
+    by a direct stakeholder in the transaction
+  - Pays out a sum equal to the price spread of the orders times the volume
+    matched to the initiator of the Match Request
+- Trade
+  - A successful trade is carried out as a result of matching two valid orders that meet
+    execution conditions.  A trade may not wholly satisfy one or both of the
+    orders in which case the affected orders are adjusted appropriately.  As
+    long as the execution requirements are met there will be an exchange of tokens
+    between accounts. Trades are recorded in a new account for posterity.
 - Investor
   - Individual investors who hold a number of tokens and wish to trade them on
     the exchange.  Investors operate as Solana thin clients who own a set of
-    accounts containing tokens and/or trade requests.  Investors post
+    exchange accounts containing tokens and/or trade requests.  Investors make
     transactions to the exchange in order to request tokens and post or cancel
     trade requests.
-- Swapper
-  - An agent who facilitates trading between investors.  Swappers operate as
-    Solana thin clients who monitor all the trade orders looking for a trade
-    match.  Once found, the Swapper issues a swap request to the exchange.
-    Swappers are the engine of the exchange and are rewarded for their efforts by
-    accumulating the price spreads of the swaps they initiate.  Swappers also
-    provide current bid/ask price and OHLCV (Open, High, Low, Close, Volume)
-    information on demand via a public network port.
+- Matcher
+  - An agent who facilitates trading between investors.  Matchers operate as
+    Solana thin clients who monitor the pool of available orders looking for matchable
```

Comment:
We can probably remove the reference to the thin client here, that is more of an implementation detail.